common/libs/VkCodecUtils/FrameProcessor.h

```diff
@@ -111,13 +111,13 @@ class FrameProcessor : public VkVideoRefCountBase {
         , m_verbose(verbose)
     {
         if (m_verbose) {
```

Comment:
If I understand correctly, with `loginfo` all the `verbose`/`m_verbose` lines are useless, and it would be better to remove them.